src/lib/containers/FluidModal.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { Button, Modal } from 'react-bootstrap'
+import { ButtonVariant } from 'react-bootstrap/esm/types'
+import Typography from '../utils/typography/Typography'
+
+type ModalAction = {
+  variant?: ButtonVariant
+  copy: React.ReactNode
+  onClick: () => void
+}
```

Comment:
"Actions" will be passed via props so the modal can enforce style guide rules like button formatting and order.